src/test/java/org/mvel2/util/StringAppenderTest.java

```diff
@@ -0,0 +1,109 @@
+package org.mvel2.util;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.mvel2.util.StringAppender;
+
+public class StringAppenderTest {
+
+  @Rule
+  public final ExpectedException thrown = ExpectedException.none();
+
+  @Test
+  public void testCharConstructor() {
+    final StringAppender stringAppender = new StringAppender('a');
+    Assert.assertEquals("", stringAppender.toString());
+    // This looks to be exhibiting a potential bug, shouldn't this return "a"?
+  }
+
+  @Test
+  public void testAppendNull() {
+    final StringAppender stringAppender = new StringAppender("a");
+    String s = null;
```

Comment:
Maybe remove this line and pass "null" as an argument directly.

Reply:
No. I need to pass it as a null string.